src/components/Issue.tsx

```diff
@@ -36,7 +45,7 @@ const Issue = ({ title, url, labels, repoUrl, date }) => {
       </h3>
       <div className='flex flex-wrap'>
         {labels.map((label) => (
-          <Label key={label.name.replaceAll(' ', '')} className='mr-1.5 mt-2'>
+          <Label key={label.name.replace(/ /g, '')} className='mr-1.5 mt-2'>
```

Comment:
@rishuxd why didn't you use replaceAll( )?

Reply:
I had this error that replaceAll() method is not available on the string type in the ES2020 version of TypeScript (that we're using), it was introduced in ES2021. That's why I used replace() with a regular expression and it did the work.

Reply:
@vatsalsinghkv, I had this error that replaceAll() method is not available on the string type in the ES2020 version of TypeScript (that we're using), it was introduced in ES2021. That's why I used replace() with a regular expression and it did the work.

Reply:
@fekete965 What do you think? should we upgrade the typescript version?

@rishuxd sorry for the delay, I'll merge the PR soon

Reply:
I'm happy to upgrade the packages if you want me to. It won't hurt to use the latest version.

Reply:
Thank you @vatsalsinghkv
Also, my suggestion too would be to upgrade the packages.

Reply:
We cannot upgrade TS, we seems to be using the latest version haha

Reply:
Not TS, Just ECMAScript

Reply:
Ah, you are referring to the `compilerOptions` it wasn't clear to me until now.